packages/replayio/src/commands/record.ts

```diff
@@ -176,3 +171,57 @@ async function record(url: string = "about:blank") {
 
   await exitProcess(0);
 }
+
+async function reportRecorderCrash(stderr: string) {
+  console.log("\nSomething went wrong while recording. Try again.");
+
+  const errorLogPath = getReplayPath("recorder-crash.log");
+  writeFileSync(errorLogPath, stderr, "utf8");
+  console.log(dim(`More information can be found in ${errorLogPath}`));
+
+  const accessToken = await checkAuthentication();
+
+  if (!accessToken) {
+    return;
+  }
```

Comment:
Some nits:

* I prefer methods like this to be in their own files (like you did with `utils/findMostRecentFile`) because IMO this makes it easier to navigate project source, but this is just a personal preference.
* I think the first few parts of this method (the console logging and `recorder-crash.log` writing) should be in the caller, rather than this method (given this method's name)

Reply:
applied both suggestions